docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,46 @@
+# Subprojects
+
+## Overview
+**Subprojects** are defined in the front-end by the Configuration module, and are stored in the `subproject` table of the database.
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.  The subproject can be used as a variable when determining which instruments are populated -- this allows for subproject-specific batteries to be defined in the `test_battery` table. 
+
+## Adding Subproject Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page. Refer to the help section of the module for further instructions on how to add or modify subprojects.
+   
+### SQL
+Subprojects can be added directly in SQL using the following command.
+
+```sql
+INSERT INTO subproject (title) VALUES('SCI');
+```
+
+
+### API
+ _not yet available. See [API documentation](../../../API/) for latest additions_
+ 
+
+## Interaction With LORIS
+
+### Projects
+**Subprojects** must be associated to Projects in order to be able to create timepoints for candidates. This association should be defined directly on the front end through the Configuration module.
+
+1. Navigate to the Configuration module
+2. In the upper left corner click on _To configure study projects click here._
+3. Under each project, you should be able to define the associated subprojects.
+
```

Comment:
clarify:  how to view the current relationships for a subproject:
```suggestion
Note: the only way to view a list of all the projects affiliated to a subproject is via the MySQL back-end. 
```

Reply:
I'm accepting this because I appreciate the time you took to contribute but it's not technically the only way. you can go on your projects and see if a specific subproject is selected.